src/marks/area.js

```diff
@@ -85,8 +92,10 @@ export class Area extends Mark {
  * [Plot.areaX](https://github.com/observablehq/plot/blob/main/README.md#plotareaxdata-options)
  * is used in the vertical orientation where the baseline and topline share *y*
  * values.
+ * @param {Data} data
+ * @param {Options} options
  */
-export function area(data, options) {
+export function area(data, options = undefined) {
```

Comment:
In order to have editors know that options is optional, we had to add these default values for options throughout. Without this, the typescript language server will consider options required:

<img width="595" alt="image" src="https://user-images.githubusercontent.com/191891/193651976-3df23e3d-d68a-4fbc-8ab2-08d540b0907d.png">


Adding ? to the @param also didn't seem to help:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/191891/193651879-7e20aebb-2a1c-4382-b3dc-36d8fe16d70d.png">

Only adding a default value will produce the ? in the signature help:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/191891/193652126-5e7d228b-e060-4498-890a-3b21cb0c6026.png">

Reply:
@mbostock pointed out that this can be done by using the proper JSDoc syntax for optional parameters (`@param {Options=} options`) 🙏